src/components/Select.vue

```diff
@@ -730,7 +744,7 @@
           }
         }
 
-        this.$emit('input', value);
+        this.$emit('update:modelValue', value);
```

Comment:
To me not sure if it’s worth it as I’m having to go through my whole app and replace a  number of instances where I’m listening to `@input` currently and replace with new `@update:modelValue` anyway as part of Vue 3 upgrade so this would just be another place to update. Maybe worthwhile if we want a vue2 compatible version or if the only place you were listening to `@input`was on vue-select?